setup.rst

```diff
@@ -62,6 +62,28 @@ to meet those requirements.
     If the installer doesn't work for you or doesn't output anything, make sure
     that the PHP `Phar extension`_ is installed and enabled on your computer.
 
+.. note::
+
+    If the SSL certificates are not properly installed in your system, you
+    may get this error:
+
+        cURL error 60: SSL certificate problem: unable to get local issuer certificate.
+
+    You can solve this issue as follows:
+
+    #. Download a file with the updated list of certificates from
+       https://curl.haxx.se/ca/cacert.pem
```

Comment:
Yes ... but this is the syntax to display links in RST:

![link-cacert-pem](https://cloud.githubusercontent.com/assets/73419/23035806/ad6a46bc-f480-11e6-822c-21dfd4a6abff.png)